README.md

```diff
@@ -10,32 +16,16 @@ This repository contains the Azure SQL bindings for Azure Functions extension co
 - **Output Binding**: takes a list of rows and upserts them into the user table (i.e. If a row doesn't already exist, it is added. If it does, it is updated).
 - **Trigger Binding**: monitors the user table for changes (i.e., row inserts, updates, and deletes) and invokes the function with updated rows.
 
-See the language specific guides for further details on support and usage of these bindings in the supported languages.
+See the language-specific guides for further details on support and usage of the Azure SQL bindings for Azure Functions in the supported languages:
 
-- [.NET](./docs/BindingsGuide_Dotnet.md)
+- [.NET (C# in-process)](./docs/BindingsGuide_Dotnet.md)
 - [Java](./docs/BindingsGuide_Java.md)
-- [Javascript](./docs/BindingsGuide_Javascript.md)
+- [NodeJS (JavaScript/TypeScript)](./docs/BindingsGuide_Javascript.md)
```

Comment:
Let's actually change this to just JS for now - this matches our learn docs and given that we don't actually have any setup/samples for TS right now it doesn't make a lot of sense to advertise that currently. 

```suggestion
- [Javascript](./docs/BindingsGuide_Javascript.md)
```

Reply:
Sounds good! I really like this new GitHub feature for small changes like this!

Reply:
Same - it makes me not feel as guilty for leaving small nit comments 😆